pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -207,4 +208,33 @@ module.exports = {
 
     assert.equal(count, otherDetails, 'Check data!');
   },
+
+  async getCellValue(rowNumber, columnNumber) {
+    const value = await I.grabTextFrom(`//tr[@data-qa="table-row"][${rowNumber}]/td[${columnNumber}]`);
+
+    return value.toLowerCase();
+  },
+
+  async checkSort(columnNumber) {
+    I.waitForVisible(this.fields.tableRow, 20);
+    const rowCount = await I.grabNumberOfVisibleElements(this.fields.tableRow);
+    let tmp;
+
+    for (i = 0; i < rowCount; i++) {
+      const cellValue = await this.getCellValue(i + 1, columnNumber);
```

Comment:
not sure if `const` makes sense here, because it will keep changing.

Reply:
`for (let i = 0; i < rowCount; i++)`

Reply:
I'm sure it does.

Reply:
> not sure if `const` makes sense here, because it will keep changing.

I'm sure it does. It's a scoped constant, so it is correct.

Reply:
> `for (let i = 0; i < rowCount; i++)`

Done, thanks.

Reply:
> not sure if `const` makes sense here, because it will keep changing.

Sure, it does )